models/articles.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchArticle = (article_id) => {
+
+    return db
+    .query(`SELECT * FROM articles
+        WHERE article_id = $1`, [article_id])
+        .then(result => {
```

Comment:
here for example a custom error might be good if the results array is empty - look up promise reject in the notes